central/vulnmgmt/service/service_impl_benchmark_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build sql_integration
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/central/testutils"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"google.golang.org/grpc"
+)
+
+func BenchmarkService_Export(b *testing.B) {
```

Comment:
These two functions seem to be essentially identical for deployments, images and vuln workloads. Do you think we could extract them as generic functions?

Reply:
I can give it a try. It probably is not as easy as it looks since the generic object would be the service and the elements that do revolve around it (service registration, client, response type for the export call).